rules-tests/CodeQuality/Rector/MethodCall/AssertCompareToSpecificMethodRector/Fixture/get_class.php.inc

```diff
@@ -27,7 +27,7 @@ final class GetClass extends \PHPUnit\Framework\TestCase
     public function test()
     {
         $something = new stdClass();
-        $this->assertInstanceOf('stdClass', $something);
+        $this->assertSame(get_class($something), 'stdClass');
```

Comment:
I think these lines can be removed completely from the tests as unrelated.

Reply:
There is a test for 
```
self::assertSame('stdClass', get_class($something));
```
being fixed by AssertCompareToSpecificMethod

And I would have consider
```
$this->assertSame(get_class($something), 'stdClass');
```
as a non-regression test that if the `get_class` call is made on the first param, nothing change.

In the same way I have added a 
```
$this->assertSame(count($something), 5);
```
as a non-regression test.

Should I remove those two non-regression test ?

Reply:
Just this line:

>  $this->assertSame(get_class($something), 'stdClass');

Reply:
I moved it in the non-regression-test `skip_first_param`